src/spectral_recovery/metrics.py

```diff
@@ -162,13 +194,8 @@ def r80p(
     return r80p_v
 
 
-@maintain_rio_attrs
```

Comment:
Are we ok no longer doing the @maintain_rio_attrs?

Reply:
We're still doing it but at a higher-level. The decorator was moved onto `compute_metrics` instead, so anything going in and out of that function will keep the rio attributes!